### ISSUE TITLE
Enable FxCop code anayzers under project properties, and add an editor.config

### ISSUE THREAD

**Issue by SimonGeering** [DevChatter/ChatterBot#42]
**Is your feature request related to a problem? Please describe.**
Ensure that all contributors to the codebase adopt a common coding style so it is easy to pick up any code in the app and work with it if someone is familiar with the codebase.

**Describe the solution you'd like**
- Enable the built-in Microsoft Rosalyn Code Analyzers for FxCop under project properties dialog. 
- Add or generate an editor.config for the solution.

**Describe alternatives you've considered**
Don't have an alternative, it would have to be a lot of effort in PR reviews!

**Additional context**
None.

**Comment by benrick** [comment]
Yep, we definitely want an editor config to make it easier for people to keep consistent code.
